MSAL/test/unit/MSALPublicClientApplicationTests.m

```diff
@@ -1295,6 +1400,46 @@ - (void)testAccountWithHomeAccountId_whenAccountExistsButNotMatching_shouldRetur
     XCTAssertNil(account);
 }
 
+- (void)testAccountWithHomeAccountId_whenFociTokenExistsForOtherClient_andAppMetadataInCache_shouldReturnAccountNoError
+{
+    //store at & rt in cache with foci flag
+    MSIDAADV2TokenResponse *msidResponse = [self msalDefaultTokenResponseWithAuthority:@"https://login.microsoftonline.com/common" familyId:@"1"];
+    MSIDConfiguration *configuration = [self msalDefaultConfigurationWithAuthority:@"https://login.microsoftonline.com/common"];
+
+    NSError *error = nil;
+    BOOL result = [self.tokenCacheAccessor saveTokensWithConfiguration:configuration
+                                                              response:msidResponse
+                                                               factory:[MSIDAADV2Oauth2Factory new]
+                                                               context:nil
+                                                                 error:&error];
+    XCTAssertTrue(result);
+    XCTAssertNil(error);
```

Comment:
nit: not sure if we need to test this here, 
1. saveTokens is already tested
2. if anything, wouldn't we better off seeing the cache result? (i.e./ count?)

Reply:
Addressed